app/src/main/res/layout-land/question_player_fragment.xml

```diff
@@ -13,7 +13,7 @@
   <FrameLayout
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/question_player_background">
+    android:background="@color/question_player_activity_layout_background_color">
```

Comment:
For this create a new color `question_player_fragment_background_color` as this is a fragment file and not an activity file. Also, removed `layout` word as this does not add any extra info.

Reply:
Done